### PR TITLE
Added a note regarding that you may have to wait before you can curl …

### PR DESCRIPTION
**Pull request by aaron-prindle**
…the Quickstart url

Taken from:
https://kubernetes.slack.com/archives/minikube/p1467142704000105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Kubernetes is available at https://192.168.99.100:443.
 
 $ kubectl run hello-minikube --image=gcr.io/google_containers/echoserver:1.4 --hostport=8000 --port=8080
 deployment "hello-minikube" created
+#NOTE: It may take a few minutes for the echoserver pod to be up and running before you can curl it
```
Comment (dlorenc) [inline README.md]: What if we change this to include a "kubectl get pods" first, so the user can see something is actually happening. Then we can make a note saying that curl won't work until the state is "Running", which can take a few minutes.

Comment (dlovitch) [inline README.md]: @dlorenc that's basically what I ended up doing. I found mention of it at [http://kubernetes.io/docs/getting-started-guides/minikube/](http://kubernetes.io/docs/getting-started-guides/minikube/) where it suggests `kubectl get pods --all-namespaces`

File: README.md [hunk 2]
```diff
@@ -43,6 +43,16 @@ Kubernetes is available at https://192.168.99.100:443.
 
 $ kubectl run hello-minikube --image=gcr.io/google_containers/echoserver:1.4 --hostport=8000 --port=8080
 deployment "hello-minikube" created
+#We have now launched an echoserver pod but we have to wait until the pod is up before curling/accessing it
+#To check whether the pod is up and running we can use the following:
+$ kubectl get pod
+NAME                              READY     STATUS              RESTARTS   AGE
+hello-minikube-3383150820-vctvh   1/1       ContainerCreating   0          3s
+#We can see that the pod is still being created from the ContainerCreating status
```
Comment (dlorenc) [inline README.md]: Small nit: Space between the comment and the text. Then LGTM.

**Comment by dlorenc** [comment]
Looks like you grabbed some extra files by accident.

**Comment by aaron-prindle** [comment]
Yes, I had an issue with corruption in my repo.  Sorting it out, will resubmit soon.

**Comment by dlorenc** [comment]
LGTM.

**Pull request merged**